code/modules/mob/living/emote.dm

```diff
@@ -209,9 +209,22 @@
 /datum/emote/living/jump
 	key = "jump"
 	key_third_person = "jumps"
-	message = "jumps!"
+	message = "jumps up and down."
 	hands_use_check = TRUE
 
+/datum/emote/living/jump/run_emote(mob/living/user, params, type_override, intentional)
+	. = ..()
+	var/turf/open/current = get_turf(user)
```

Comment:
Does this work from inside a locker? _Should_ it work from inside a locker?
One of those cases where `user.loc` might actually be more appropriate.

Reply:
I didn't test that, but no reason it wouldn't work from inside a locker. As for whether it should, does it enhance or detract from the comedy if someone bounces a locker up and down to drop it through the floor?